os/net/socket/bsd_socket_api.c

```diff
@@ -132,7 +132,7 @@ int listen(int s, int backlog)
 	return lwip_listen(s, backlog);
 }
 
-int recv(int s, void *mem, size_t len, int flags)
+ssize_t recv(int s, void *mem, size_t len, int flags)
```

Comment:
I can't match your commit title and this change. When you change this, can we run tls selftest?

Reply:
Isn't it a change to make POSIX-compatible?

Reply:
@sunghan-chang 
- Yes, if we enable TLS option in menuconfig, build errors occur. So I modified...
- I think this change is POSIX-compatible.
  (https://pubs.opengroup.org/onlinepubs/9699919799/functions/send.html)

Reply:
That's why I left this comment. You say `enable tls sefltest` through title but your change says `modify this for POSIX-compatible`. 
And is this fixing build breaks or enabling tls selftest? I can't find any change which shows enabling.